packages/server/modules/workspaces/tests/unit/services/join.spec.ts

```diff
@@ -0,0 +1,137 @@
+import { UserEmail } from '@/modules/core/domain/userEmails/types'
```

Comment:
Didn't we agree to colocate unit tests with the files they're testing?

Reply:
It was trialed, but we def didn't land on this being the new standard. I don't think moving a brownfield project to this new pattern would not make things more confusing, having old tests in a separate folder and new tests next to the file.